PENDING_RELEASE_NOTES.md

```diff
@@ -14,6 +14,7 @@ _Date TBD_
   * Update chart data generation application
   * Add API endpoint to retrieve chart data
 * Fixed duplicate CHPL Product Number issue when editing a product and merging developers
+* Updated the XML and CSV file creation batch process to retrieve data asynchronously
```

Comment:
Should be in "PENDING_PENDING_RELEASE_NOTES" file (as well as above line. This file is for what's actually releasing in the next release, and this story isn't going until the one after